appserver/tests/tck/microprofile/pom.xml

```diff
@@ -19,36 +19,44 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
+     
     <parent>
-        <groupId>org.glassfish.main.tests.tck</groupId>
-        <artifactId>tck</artifactId>
-        <version>8.0.0-SNAPSHOT</version>
+        <groupId>org.eclipse.ee4j</groupId>
```

Comment:
Why?

Reply:
I was trying to run the TCK against arbitirary GlassFish versions. With the project parent it insisted upon having basically all of GlassFisg build in the version that the project happened to be. This is unnecessary when testing with such specific version.